tasktiger/stats.py

```diff
@@ -64,9 +78,34 @@ def compute_stats(self) -> None:
                     utilization=utilization,
                 )
 
+    def start_logging_thread(self) -> None:
+        if not self._logging_thread:
+            self._logging_thread = StatsLoggingThread(self)
+            self._logging_thread.start()
+
+    def stop_logging_thread(self) -> None:
+        if self._logging_thread:
+            self._logging_thread.stop()
+            self._logging_thread = None
+
+    def _record_time_busy(self, now: float) -> None:
+        assert self._task_start_time
+        self._time_busy += now - max(self._task_start_time, self._time_start)
+
+
+class StatsLoggingThread(threading.Thread):
+    def __init__(self, stats: Stats) -> None:
+        super().__init__()
+
+        self.tiger = stats.tiger
```

Comment:
Should this ideally just say `self._stats_interval = stats.tiger.config["STATS_INTERVAL"]` rather than looking it up every time?